litellm/model_prices_and_context_window_backup.json

```diff
@@ -591,15 +591,36 @@
     },
     "mistral/mistral-small": {
         "max_tokens": 8192,
-        "input_cost_per_token": 0.00000066,
-        "output_cost_per_token": 0.00000197,
+        "input_cost_per_token": 0.000002,
+        "output_cost_per_token": 0.000006,
+        "litellm_provider": "mistral",
+        "mode": "chat"
+    },
+    "mistral/mistral-small-latest": {
+        "max_tokens": 8192,
+        "input_cost_per_token": 0.000002,
+        "output_cost_per_token": 0.000006,
         "litellm_provider": "mistral",
         "mode": "chat"
     },
     "mistral/mistral-medium": {
         "max_tokens": 8192,
-        "input_cost_per_token": 0.00000273,
-        "output_cost_per_token": 0.00000820,
+        "input_cost_per_token": 0.0000027,
+        "output_cost_per_token": 0.0000081,
```

Comment:
this difference was coming from the conversion error from euro -> dollar most likely.
initially prices were expressed just in Eur -> https://web.archive.org/web/20240103230150/https://docs.mistral.ai/platform/pricing/
and 2.5 eur ~= 2.72 $